Mathlib/Analysis/Convex/Between.lean

```diff
@@ -70,8 +70,11 @@ theorem right_mem_affineSegment (x y : P) : y ∈ affineSegment R x y :=
 theorem affineSegment_same (x : P) : affineSegment R x x = {x} := by
   -- porting note: added as this doesn't do anything in `simp_rw` any more
   rw [affineSegment]
+  -- Note: when adding "simp made no progress" in lean4#2336,
+  -- somehow this `simp` doesn't work in `simp_rw`.
+  simp only [lineMap_same]
```

Comment:
Interestingly it seems like `simp_rw` does work but it also throws the error:

The goal after this `simp only` is `(fun a ↦ ↑(const R R x) a) '' Set.Icc 0 1 = {x}`.

If I replace it with
```
skip
simp_rw [lineMap_same]
```
then having the cursor before skip says `↑(lineMap x x) '' Set.Icc 0 1 = {x}` but having it before `simp_rw` says `(fun a ↦ ↑(lineMap x x) a) '' Set.Icc 0 1 = {x}`, i.e. the same state you get with the `simp only`.